heron/examples/src/java/com/twitter/heron/examples/CustomGroupingTopology.java

```diff
@@ -18,17 +18,17 @@
 import java.util.List;
 import java.util.Map;
 
-import backtype.storm.Config;
-import backtype.storm.StormSubmitter;
-import backtype.storm.generated.GlobalStreamId;
-import backtype.storm.grouping.CustomStreamGrouping;
-import backtype.storm.task.OutputCollector;
-import backtype.storm.task.TopologyContext;
-import backtype.storm.task.WorkerTopologyContext;
-import backtype.storm.topology.OutputFieldsDeclarer;
-import backtype.storm.topology.TopologyBuilder;
-import backtype.storm.topology.base.BaseRichBolt;
-import backtype.storm.tuple.Tuple;
+import org.apache.storm.Config;
```

Comment:
@objmagic you can do that separately, merge and do this review on top of it. I'm not strictly against that (maybe too much action needed) but I will prefer to do things this way.